acceptance/package.json

```diff
@@ -7,7 +7,15 @@
     "build": "cp ../dist/iopipe.js ./iopipe.js",
     "deploy": "serverless deploy",
     "invoke": "sls invoke -f simpleSuccess",
-    "local": "sls invoke local -f simpleSuccess",
-    "all": "npm run build && npm run local && npm run deploy"
+    "localCallback": "sls invoke local -f callback",
+    "localContextSuccess": "sls invoke local -f contextSuccess",
```

Comment:
No, because I don't want to deal with non-0 exit for sls local invoke. If you can come up with something that works then I'll include it